### ISSUE TITLE
Singularity image pull: "FATAL:   the requested image was not found"

### ISSUE THREAD

**Issue by CSree** [icbi-lab/nextNEOpi#78]
<img width="878" alt="Screenshot 2024-07-09 at 8 51 01 AM" src="https://github.com/icbi-lab/nextNEOpi/assets/67707774/07bdf62c-8050-40aa-9837-82789c472d8a">
[slurm-38619545.txt](https://github.com/user-attachments/files/16143602/slurm-38619545.txt)
Hi,
I ran the pipeline using -profile singularity , and got the following error:

Pulling Singularity image https://apps-01.i-med.ac.at/images/singularity/nextNEOpi_1.3.2_18734d43.sif [cache /gpfs/data/sequence/results/krogsgaardlab/2024-04-23/results_NeoagPipeline_2024/Sample18/work/singularity/apps-01.i-med.ac.at-images-singularity-nextNEOpi_1.3.2_18734d43.sif]
Pulling Singularity image https://depot.galaxyproject.org/singularity/gatk:3.8--hdfd78af_11 [cache /gpfs/data/sequence/results/krogsgaardlab/2024-04-23/results_NeoagPipeline_2024/Sample18/work/singularity/depot.galaxyproject.org-singularity-gatk-3.8--hdfd78af_11.img]
INFO: Mutect1 not available, skipping....
Pulling Singularity image https://depot.galaxyproject.org/singularity/cnvkit:0.9.9--pyhdfd78af_0 [cache /gpfs/data/sequence/results/krogsgaardlab/2024-04-23/results_NeoagPipeline_2024/Sample18/work/singularity/depot.galaxyproject.org-singularity-cnvkit-0.9.9--pyhdfd78af_0.img]
WARN: Singularity cache directory has not been defined -- Remote image will be stored in the path: /gpfs/data/sequence/results/krogsgaardlab/2024-04-23/results_NeoagPipeline_2024/Sample18/work/singularity -- Use env variable NXF_SINGULARITY_CACHEDIR to specify a different location
Error executing process > 'RegionsBedToIntervalList (RegionsBedToIntervalList)'

Caused by:
  Failed to pull singularity image
  command: singularity pull  --name apps-01.i-med.ac.at-images-singularity-nextNEOpi_1.3.2_18734d43.sif.pulling.1719238039463 https://apps-01.i-med.ac.at/images/singularity/nextNEOpi_1.3.2_18734d43.sif > /dev/null
  status : 255
  message:
    INFO:    Downloading network image
    FATAL:   the requested image was not found

The images stored in the temporary location are similar to those generated during previous successful runs of the pipeline. However this time, the images are not found. 
Thanks in advance for your help
Chai

**Comment by riederd** [comment]
Hi, we (accidentally) removed images for the outdated version, I put them back on the server. I hope they are the correct ones.

However, we strongly suggest to use a more recent version of `nextNEOpi` (e.g. v1.4.1)

**Comment by CSree** [comment]
Many thanks for your prompt response! I will update with the latest, and run it again.
Have a great day

**Comment by CSree** [comment]
Hi, 
I have 2 issues , while running
1. the Singularity error is still thrown, albeit after 7hrs of pipeline running, as opposed to after 17 mins previously

<img width="757" alt="Screenshot 2024-08-09 at 12 04 19 PM" src="https://github.com/user-attachments/assets/7bf0bb94-2bfd-4583-9a04-a2416f2f3aa9">

2. I downloaded the latest license, however still it is not recognized by the pipeline.

<img width="646" alt="Screenshot 2024-08-09 at 12 12 52 PM" src="https://github.com/user-attachments/assets/11292dae-73b0-40eb-9e51-bea43e79cbe2">

So I did run with tcr - False.

thanks in advance 
Chai
